### ISSUE TITLE
Task Limit

### ISSUE THREAD

**Issue by Gromith** [janvv/outlook-taskboard#28]
Can anyone confirm that the limit function works in this version? I have set a limit, but i can still move tasks into the column.

**Comment by janvv** [comment]
I'll check it. Only after the XMas holidays.

**Comment by janvv** [comment]
Oh my... just tested it, you are right. What is Kanban without a WIP limit? I am going to fix it.

**Comment by janvv** [comment]
Fixed

**Comment by Gromith** [comment]
Hi janvv, thanks for this fix but it seems the limit is not working properly yet. With a limit of 2 you can push 3 tasks into the columns. No matter what Limit you set, you can push exactly one task more in the columns. Tested with clean install.

**Comment by janvv** [comment]
Hmm... I tested with a limit of 2 in both columns and it seems to work perfectly:
![image](https://user-images.githubusercontent.com/4539260/34653898-a2362bee-f3f3-11e7-9aab-76f2de1b4ffb.png)
How do you add the third task?

**Comment by Gromith** [comment]
I created all tasks in the backlog. The issue is only in the NEXT column. IN PROGRESS works as expected.
How do you create this gif?

**Comment by Gromith** [comment]
i think i found another issue:

![dragndrop_issue](https://user-images.githubusercontent.com/34379664/34654212-f7cb4896-f3f8-11e7-908c-eb803b5e918e.gif)

throwing a task into outside of a column was not a problem before.

**Comment by janvv** [comment]
I fixed the error message when dropping outside one of the columns. I also implemented another fix to avoid a list going beyond the limit. I am not yet happy with the way I implemented it, but it works. A better implementation needs some more time and testing, so that's on my backlog for later this week.
Please let me know if this version works for you.

**Comment by Gromith** [comment]
Hi janvv,
I can confirm that the error message no longer appears. Many thanks!
The bug with de Limit is still there

**Comment by janvv** [comment]
Please test the Limit again

**Comment by Gromith** [comment]
Hi janvv,
now it works as expected. Many Thanks!!
